README.rst

```diff
@@ -18,6 +18,12 @@ IP blacklist is managed by properties of ``blacklist`` key inside ``configuratio
   Example: given the category ``test``, a file named ``/usr/share/nethserver-blacklist/ipsets/test.netset`` or ``/usr/share/nethserver-blacklist/ipsets/test.ipset`` must exist
 * ``Url``: the GIT URL from where blacklists will be downloaded
 * ``Whitelist``: a comma-separated list of hosts excluded from the blacklists. The host can be an IP, a CIDR, an host object or a CIDR object
+* ``MaxElem`` : It does define the maximal number of elements which can be stored in the set, default 131072. you have to reboot the server ater a change, a set of ipset cannot be changed on the fly. Shorewall at the server boot will create the set automatically.
+
+::
+
+ config setprop blacklist MaxElem 262144
+ reboot
```

Comment:
I definitely do not like this :D

Could you provide a better command, or a combination, to avoid the reboot?
If you have troubles, I can try to do it.

Just let me know ;)

Reply:
we have to delete a set to change the maxelem value, except if I am wrong of course

Reply:
Yes, you need to delete all sets and re-create them

Reply:
so either we add a complex code to destroy the set and create it again, or we are pragmatic, reboot the server

Reply:
I propose to stop firewall, delete the sets, then start shorewall

```
 config setprop blacklist MaxElem 262144
 systemctl stop shorewall
 for S in $(ipset -L -name | grep '^bl-'); do : ; ipset destroy  $S; done
 systemctl start shorewall
```

we could even implement either an event, or directly use `/etc/shorewall/stopped` to destroy the set and created on the fly when shorewall starts.

Reply:
That's a very good solution!
Adding it to the README, is enough for now :)

Reply:
done please review @gsanchietti